src/App/Fossa/FossaAPIV1.hs

```diff
@@ -457,15 +458,17 @@ getIssues apiOpts ProjectRevision{..} = fossaReq $ do
 
 ---------------
 
-attributionEndpoint :: Url 'Https -> Int -> Locator -> Url 'Https
-attributionEndpoint baseurl orgId locator = baseurl /: "api" /: "revisions" /: renderLocatorUrl orgId locator /: "attribution" /: "json"
+attributionEndpoint :: Url 'Https -> Int -> Locator -> ReportOutputFormat -> Url 'Https
+attributionEndpoint baseurl orgId locator ReportJson = baseurl /: "api" /: "revisions" /: renderLocatorUrl orgId locator /: "attribution" /: "json"
```

Comment:
This feels really verbose because it repeats the whole URL for all the options.  Could we just make a `where` expression that does a `case` on the format to generate that path segment?